include/highfive/bits/H5Utils.hpp

```diff
@@ -37,10 +37,15 @@ template <std::size_t N>
 class FixedLenStringArray;
 
 
+template <typename T>
+using unqualified_t = typename std::remove_const<typename std::remove_reference<T>::type
+        >::type;
```

Comment:
That looks…weird

Reply:
looks fine to me?

Reply:
with C++20: https://en.cppreference.com/w/cpp/types/remove_cvref

Reply:
LGTM as well

Reply:
Well, I hear complaints about
```
):
```
In Python code… so `>::type;` looks just as weird to me :)